### DISCUSSION TITLE
History autosuggest in tmux?

### DISCUSSION THREAD

**Discussion by Yomgui33** [ThePorgs/Exegol#145] [Q&A] [answered]
The history-based autosuggestions provided by Exegol are super helpful, but I lose them as soon as I open a tmux session. 

Any idea how to have the same autosuggestions inside tmux? 

Thanks & cheer!

**Comment by Dramelac** [comment]
Hello,

Iindeed I have the same problem with tmux but since I use mostly `CTRL+R` and it works even in tmux it did not bother me more than that and I didn't take the time to investigate but if someone finds the right configuration it would be awesome!

**Comment by Yomgui33** [comment, marked as answer]
I *finally* found what causes the issue while searching to solve another problem: the cursor kept disappearing in tmux after certain commands, but not with newly installed bins (I used xclip to investigate). 

Typing `evil-winrm` then space in tmux? No more cursor. 
Typing `xclip` then space? Cursor was still there, blinking. 
And no more blinking issues if using bash inside tmux instead of zsh.

The source of both problems is in .zshrc :   

`ZSH_AUTOSUGGEST_HIGHLIGHT_STYLE='fg=626262'`

This color seems to make the auto-suggestion "invisible" (same as background color maybe?) due to the whole color scheme (at least for me).

If you replace the color with something else (I used 60), then you have the auto-suggestions from history AND no more cursor issues (it wasn't blinking because there was some "invisible" suggestion from history).
